src/coreclr/src/utilcode/ccomprc.cpp

```diff
@@ -541,17 +545,59 @@ HRESULT CCompRC::LoadResourceFile(HRESOURCEDLL * pHInst, LPCWSTR lpFileName)
 #ifdef HOST_WINDOWS
```

Comment:
Would it make sense to make both Windows and Unix paths use the new PEDecoder path and turn this function platform agnostic?

Reply:
String resouces packaged as PE files exists only on Windows. There is no point in trying to make it cross-platform.

Reply:
The callsites of this function are in exceptional paths, maybe we can restrict the call sites for Windows and ifdef out this function for Windows-only; as a permanent solution?

Reply:
Thanks, added `ifdef`s around `LoadResourceFile` and its usages. Old commits that implemented it were 31f1fe4501506b1db343f85fb0ed6290390eeaf8 and 7662f49 (if we every want to revisit it in the future).